### PR TITLE
Add uhura main program.

### PR DESCRIPTION
**Pull request by keep94**

### CODE REVIEW

File: cmd/uhura/main.go [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+	"log"
+	"net/http"
+	"net/rpc"
+	"net/url"
+	"path"
+	"time"
+)
+
+var (
+	fPort      = flag.Int("port", 4242, "port")
```
Comment (rgooch) [inline cmd/uhura/main.go]: How about using "portNum" as the flag name? That would make it consistent with **Dominator** daemons.
Also, how about picking a port in the 69xx range? Maybe 691x?

Comment (keep94) [reply cmd/uhura/main.go]: Will change to portNum, but the opentsdb standard port is 4242.

Comment (rgooch) [reply cmd/uhura/main.go]: Right, OK.

**Comment by keep94** [comment]
Ok -port changed to -portNum

**Pull request merged**